src/puppet-wechat4u.ts

```diff
@@ -142,6 +135,38 @@ export class PuppetWechat4u extends PUPPET.Puppet {
     }
   }
 
+  /**
+   * 由于获取用户信息目前是发现一个不存在的就会调用一次，如果是群消息就会发出特别多的信息请求，最后就是大部分接口请求失效
+   * 目前采用定时器的方式去定时获取用户信息
+   * 1、当第一次触发搜索请求的时候会开启一个定时器
+   * 2、所有请求的未知用户id都会存储到unknownContactId中
+   * 3、定时器会在500ms执行一次，每次取unknownContactId中的50条数据
+   * 4、如果unknownContactId的数据为空就会清空定时器，等待下一次建立
```

Comment:
I'd love to suggest that we should keep using English as the primary language in the source code as well as the comments because of the maximum capabilities for all developers' reading and understanding.

Reply:
I have created another PR to complete this question